gradle-scalatest.plugin

```diff
@@ -0,0 +1,21 @@
+buildscript {
+	repositories {
+		maven {
+			url "https://plugins.gradle.org/m2/"
+		}
+	}
+	dependencies {
+		classpath "com.github.maiflai:gradle-scalatest:0.8"
+	}
+}
+
+if (!project.plugins.findPlugin(com.github.maiflai.ScalaTestPlugin)) {
```

Comment:
I agree with tyler that we should go with conventional first. Looking back at SBT problems we've had, I don't want to do anything that would cause confusion for future maintainers.

In other words, if the guard (if-statement) is functional, then comment when it can be removed, hopefully via a github issue being resolved.